### PR TITLE
Fix WG June 20th video call agenda link

### PR DESCRIPTION
**Pull request by chicoxyzzy**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -61,4 +61,4 @@ Meeting process is documented:
    * [CG May 29th video call](2018/CG-05-29.md)
    * [WG June 6th video call](2018/WG-06-06.md)
    * [CG June 12th video call](2018/CG-06-12.md)
-   * [WG June 20th video call](2018/WG-06-020.md)
+   * [WG June 20th video call](2018/WG-06-02.md)
```
Comment (binji) [inline README.md]: Thanks but should be 06-20! :-)

Comment (chicoxyzzy) [reply README.md]: oops. yes, I'll fix this =)

Comment (chicoxyzzy) [reply README.md]: fixed

**Comment by binji** [comment]
Thanks!

**Pull request merged**